script.js

```diff
@@ -0,0 +1,409 @@
+alert("Welcome to the kingdom of Erebor!");
+
+// choose characters
+var start;
+
+// for the hobbit
+var fightOrHome;
+
+// for the dwarf
+var honorOrTreasure;
+
+// for the elf
+var daleOrMirkwood;
+
+// how many times player loses gandalf game
+var loseCounter = 0;
+
+// to restart the game
+	var playAgain = function() {
+
+	var restart = prompt("Would you like to play again? (Yes/No)").toLowerCase();
+	if (restart == "yes") {
```

Comment:
I would write it this way, rather than chaining functions.

var restart = prompt("Would you like to play again? (Yes/No)");
	if (restart.toLowerCase == "yes") {